emctl/cmd/client/command/mesh_reset.go

```diff
@@ -42,6 +43,7 @@ func reset(cmd *cobra.Command, resetFlags *flags.Reset) {
 	}
 
 	clearFuncs := []installation.ClearFunc{
+		shadowservice.Clear,
```

Comment:
I think we could support the `--only-clear-addon` option.

Reply:
added the same flags `--only-add-on` and `--add-ons=` to the `reset` command.